### PR TITLE
l10n: uk: update translation

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -8528,40 +8528,40 @@ msgstr "не вдалося очистити \"%s\""
...
 #: builtin/gc.c:2534
 msgid "neither systemd timers nor crontab are available"
-msgstr ""
+msgstr "недоступні ні systemd таймери, ні crontab"
 
 #: builtin/gc.c:2549
 #, c-format
 msgid "%s scheduler is not available"
-msgstr ""
+msgstr "%s планувальник %s недоступний"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "%s планувальник недоступний"
```

File: po/uk.po [hunk 2]
```diff
@@ -8595,204 +8595,204 @@ msgstr "не вдалося прочитати дерево (%s)"
...
 
 #: builtin/grep.c:964
 msgid "show filename only once above matches from same file"
-msgstr ""
+msgstr "показувати назву файлу лише один раз над збігами з того файлу"
 
 #: builtin/grep.c:967
 msgid "show <n> context lines before and after matches"
-msgstr ""
+msgstr "показувати <н>рядків до і після збігу"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "показувати <н> рядків до і після збігу"
```

File: po/uk.po [hunk 3]
```diff
@@ -8822,120 +8822,126 @@ msgstr "вказано неприпустиму кількість потокі
...
 #: builtin/help.c:96
 msgid "git help [[-i|--info] [-m|--man] [-w|--web]] [<command>|<doc>]"
-msgstr ""
+msgstr "git help [[-i|--info] [-m|--man] [-w|--web]] [<команда>|<док>]"
 
 #: builtin/help.c:215
 #, c-format
 msgid "unrecognized help format '%s'"
-msgstr ""
+msgstr "нерозпізнаний формат довідки '%s'"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "нерозпізнаний формат довідки \"%s\""
```

File: po/uk.po [hunk 4]
```diff
@@ -8948,26 +8954,30 @@ msgid ""
...
 
 #: builtin/help.c:503
 msgid "no man viewer handled the request"
-msgstr ""
+msgstr "жоден з переглядачів керівництва користувача не обробив запит"
 
 #: builtin/help.c:510
 msgid "no info viewer handled the request"
-msgstr ""
+msgstr "жоден з переглядачів керівництва користувача не обробив запит"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "жоден з переглядачів інформації не обробив запит"
```
🤷🏻‍♀️

**Review by kasya — COMMENTED** [review]
💪🏼

File: po/uk.po (hunk 1 above)
Comment (arkid15r) [reply po/uk.po]: 👍

**Review by arkid15r — COMMENTED** [review]
Great review w/ attention to details!

**Pull request merged**